### DISCUSSION TITLE
Computation of total oil mass

### DISCUSSION THREAD

**Discussion by FrancoisDuquesne** [OpenDrift/opendrift#839] [Q&A] [answered]
Hello,  

I was able to release on multiple times and locations using vectors as input.
However, I have an issue with the computation of the total oil mass when using a list as time release.

Digging into your code I found that the total duration is computed in openoil.py l1562 as `(time[1] - time[0])`. This does not work if you give more than two dates in the release time list. Is this an expected behavior? Should the list only count two dates? If not then the l1562 should read `(time[-1] - time[0])`

Kind regards.

**Comment by knutfrode** [comment, marked as answer]
Hi,

It is not well documented, but time must be either a single time, or a two-element list `[start_time, end_time] `
For your case of multiple points, seeding has to be done in a loop:
```
for lon,lat,time in zip(lons, lats, times):
    o.seed_elements(lon=lon, lat=lat, time=time, <additional arguments such as number and m3_per_hour>)
```

**Comment by knutfrode** [comment]
Btw, you may use Issues instead of Discussions for questions like this. I am now converting this to an Issue.
